roles/openshift_register_nodes/tasks/main.yml

```diff
@@ -19,7 +19,7 @@
 - name: Create node config
   command: >
     /usr/bin/openshift admin create-node-config
-      --node-dir={{ openshift_cert_dir }}/node-{{ item.openshift.common.hostname }}
+      --node-dir={{ openshift_master_config_dir }}/node-{{ item.openshift.common.hostname }}
```

Comment:
Yeah, I thought about moving them to `/etc/openshift/nodes` (pluralized) but I thought that was a bit unclear. I think it may be problematic to leave it under master as I think they intend to bundle that tree up into secrets so agree with moving it. I like `/etc/openshift/generated-node-configs` best.